test/render/test-cases.js

```diff
@@ -1121,5 +1124,44 @@ export const TEST_CASES = [
       })
     ],
     goldenImage: './test/render/golden-images/contour-isobands-lnglat.png'
+  },
+  {
+    name: 'h3-hexagon-layer',
+    viewState: {
+      latitude: 37.78,
+      longitude: -122.45,
+      zoom: 11,
+      pitch: 30,
+      bearing: 0
+    },
+    layers: [
+      new H3HexagonLayer({
+        data: h3.kRing('882830829bfffff', 4),
+        getHexagon: d => d,
+        getColor: (d, {index}) => [255, index * 5, 0],
+        getElevation: (d, {index}) => index * 100
```

Comment:
is it possible to add another `getCoverage` accessor?

Reply:
Yes? What is the use case, do we need different coverage for each hexagon?

Reply:
As shown here https://twitter.com/carnby/status/1059530716269154306
It's useful for odt analysis

kepler.gl currently edited shader to enable `instanceCoverage`
https://github.com/uber/kepler.gl/blob/master/src/deckgl-layers/h3-hexagon-cell-layer/h3-hexagon-cell-layer.js#L26-L41

Reply:
Will do in a follow up PR.